### ISSUE TITLE
`no such option: "--integration"` when using `v0` tag

### ISSUE THREAD

**Issue by mateuszkwiecinski** [emulator-wtf/run-tests#2]
Comparing https://github.com/mateuszkwiecinski/lottie-android/blob/8d6e7c212b3885b39b49f9968af3a84b6091757b/.github/workflows/validate.yml#L80C6-L88 with the sample from Readme: https://github.com/emulator-wtf/run-tests these two look pretty the same. Despite that the build fails with `no such option: "--integration"`:

![image](https://github.com/emulator-wtf/run-tests/assets/36954793/22feb57b-b2d3-4762-9f6d-bb65580ce2e8)

\+ then
```
Warning: ew-cli invoke failed: Error: The process '/home/runner/.cache/emulator-wtf/bin/ew-cli' failed with exit code 1
Error: Error: The process '/home/runner/.cache/emulator-wtf/bin/ew-cli' failed with exit code 1
```

It looks like the `--integration` argument added week ago is not available with the default setup https://github.com/emulator-wtf/invoke/blob/19541297ededf0812afa9cc061ee7f199ec23f44/src/index.ts#L169

https://docs.emulator.wtf/changelog/ doesn't mention any extra steps needed to use the github-action integraiton

**Comment by madisp** [comment]
hey, thanks for the report! indeed, looks like the current action combination is borken, we're on it

**Comment by madisp** [comment]
@mateuszkwiecinski this should be fixed now with `run-tests@v0` or `run-tests@v0.9.5`. We recommend to use the latter if you want the builds to be 100% deterministic.
